app/src/hooks/balance/useBalance.ts

```diff
@@ -10,21 +10,30 @@ const defaultValue: TokenBalance[] = tokenList.map((t) => ({
   balance: BigNumber.from(0)
 }))
 
-const useBalance = (orgId = 0) => {
+interface Props {
+  orgId?: number
+  address?: string
+}
+
+const useBalance = (orgId = 0, address = '') => {
```

Comment:
since this accepts 2 parameters, can we split this into 2 hooks?

`const useBalance = (address) => {` and `const useOrganizationBalance = (orgId = 0) => {`